utils/hotspot-cli/src/main.rs

```diff
@@ -0,0 +1,438 @@
+    let (dao, _dao_bump) = Pubkey::find_program_address(
+        &[
+            "dao".as_bytes(),
+            Pubkey::from_str(HNT_MINT).unwrap().as_ref(),
+        ],
+        &HEM_PID,
+    );
+    let (key_to_asset, _kta_bump) =
+        Pubkey::find_program_address(&["key_to_asset".as_bytes(), dao.as_ref(), &hash], &HEM_PID);
+
+    let kta = program.rpc().get_account(&key_to_asset).unwrap();
+    // If the entity has not been issued, issue it. Otherwise, onboard it.
+    if kta.data.is_empty() {
+        // construct the issue entity transaction
+        let issue_entity_accounts = construct_issue_entity_accounts(
+            &program,
+            Pubkey::from_str(HNT_MINT).unwrap(),
+            &entity_key,
+        );
+        let compute_ix = ComputeBudgetInstruction::set_compute_unit_limit(200000);
+        let tx = program
+            .request()
+            .args(helium_entity_manager::instruction::IssueDataOnlyEntityV0 {
+                args: IssueDataOnlyEntityArgsV0 {
+                    entity_key: entity_key.clone(),
+                },
+            })
+            .accounts(issue_entity_accounts)
+            .instruction(compute_ix)
+            .signed_transaction()
+            .unwrap();
+
+        let serialized_tx = hex::encode(&bincode::serialize(&tx).unwrap());
+        // verify the base64 transaction with the ecc-sig-verifier
+        let url = env::var("ECC_VERIFIER_URL").expect("ECC_VERIFIER_URL must be set");
+
+        let req_client = reqwest::Client::new();
+        let response = req_client
+            .post(url)
+            .body(
+                serde_json::to_string(&VerifyRequest {
+                    transaction: &serialized_tx,
+                    msg: &args.transaction,
+                    signature: &args.signature,
+                })
+                .unwrap(),
+            )
+            .send()
+            .await
+            .unwrap()
+            .json::<VerifyResponse>()
+            .await
+            .unwrap();
+        let raw_signed_tx = hex::decode(response.transaction).unwrap();
+        let signed_tx: Transaction = bincode::deserialize(&raw_signed_tx).unwrap();
+
+        program
+            .rpc()
+            .send_and_confirm_transaction_with_spinner(&signed_tx)
+            .unwrap();
+    } else {
+        println!("Entity already issued");
+    }
+
+    println!("Onboarding hotspot");
+
+    let kta_raw = program.rpc().get_account(&key_to_asset).unwrap();
+    let kta_acc = KeyToAssetV0::try_deserialize(&mut kta_raw.data.as_slice()).unwrap();
+    let issue_entity_accounts =
+        construct_onboard_iot_accounts(&program, Pubkey::from_str(HNT_MINT).unwrap(), &entity_key);
+    let compute_ix = ComputeBudgetInstruction::set_compute_unit_limit(200000);
+
+    let req_client = reqwest::Client::new();
+    let get_asset_response = req_client
+        .post(program.rpc().url())
+        .body(format!(
```

Comment:
use the `json` feature on reqwest and use `json(...)` here? that'll make content types set correctly too